src/WidgetMessaging.js

```diff
@@ -40,6 +45,7 @@ export default class WidgetMessaging {
         this.target = target;
         this.fromWidget = global.mxFromWidgetMessaging;
         this.toWidget = global.mxToWidgetMessaging;
+        this._openIdHandlerRef = this._onOpenIdRequest.bind(this);
```

Comment:
Nit: I think it's more typical to save the bound version with the same name as the function, as in:

```suggestion
        this._onOpenIdRequest = this._onOpenIdRequest.bind(this);
```

Reply:
will apply manually